Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -1222,6 +1222,10 @@
   #define TFT_DEFAULT_ORIENTATION (TFT_EXCHANGE_XY)
   #define TFT_RES_480x320
   #define TFT_INTERFACE_SPI
+#elif ENABLED(LERDGE_TFT35)         // ST7796
```

Comment:
Marlin won't see `LERDGE_TFT35` defined in a pins file, so there is no point in defining it there.